source/stylesheets/modules/_toc.scss

```diff
@@ -44,3 +44,7 @@
     display: none;
   }
 }
+
+.toc-link--in-view {
+  font-weight: bold;
```

Comment:
Can we change the hover state to be like [the prototype](https://govuk-product-page-prototypes.herokuapp.com/govuk-paas/documentation) – inverse white on blue, and lose the bold?